get-involved.md

```diff
@@ -4,46 +4,47 @@ title: Get Involved
 order: 2
 ---
 
-Interested? You can start working with us in a few quick steps:
+Start by checking out our [Vision]({{ site.baseurl }}/about) and [Code of Conduct]({{ site.baseurl }}/code-of-conduct), then:
 
-1. Come say hi on our {% include icon-matrix.html %} chat
-  - Create an account to join `#tomesh` and introduce yourself on our `#welcome` channel!
-  - If you plan on connecting via a [mobile app](https://riot.im/), make sure you set the homeserver to `https://matrix.tomesh.net` under 'Use custom server options (advanced)'.
+1. Say hi on our chat {% include icon-matrix.html %}
+    - Create an account to join `#tomesh` and introduce yourself on our `#welcome` channel!
+    - If you connect via the [Riot mobile app](https://riot.im/), select _Use custom server options (advanced)_ and set the homeserver to `https://matrix.tomesh.net`
 
-2. Check out our [vision]({{ site.baseurl }}/about) and [Code of Conduct]({{ site.baseurl }}/code-of-conduct)
```

Comment:
Why did you cut this out? I think this is pretty good way to cue that we have a code of conduct